chsdi/templates/htmlpopup/hochwassergrenzwertpegel.mako

```diff
@@ -0,0 +1,72 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-hochwassergrenzwertpegel.name')}</td>
+      <td>${c['attributes']['name'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-hochwassergrenzwertpegel.einzugsgebietsflaeche')}</td>
+      <td>${round(c['attributes']['einzugsgebietsflaeche'],2) or '-'}</td>
```

Comment:
If attribute is None, what will happen with `round(None)`. Maybe something like `${round(c['attributes']['einzugsgebietsflaeche'],2) if round(c['attributes']['einzugsgebietsflaeche'] else '-'}` is a little more secure.